memcached.c

```diff
@@ -166,7 +168,16 @@ static rel_time_t realtime(const time_t exptime) {
     }
 }
 
+static void stats_report_latency(stat_cmd_t cmd, hires_time_t latency) {
+    if (cmd < STAT_CMD_COUNT) {
+        STATS_LOCK();
+        percentile_sampler_record_sample(&stats.per_cmd_stats[cmd].handle_latency, latency);
+        STATS_UNLOCK();
+    }
+}
+
 static void stats_init(void) {
+    size_t i;
```

Comment:
`i` is being compared against `sizeof(stats.per_cmd_stats)/sizeof(*stats.per_cmd_stats)` which is a `size_t`. This ensures no platform-dependent compiler warnings with differences between `size_t` and `int` (or whatever integral type is selected), such as signedness or bit-width.